manifests/job/present.pp

```diff
@@ -78,7 +79,7 @@
   exec { "jenkins update-job ${jobname}":
     command => "${cat_config} | ${update_job}",
     onlyif  => "test -e ${config_path}",
-    unless  => "diff -b -q ${config_path} ${tmp_config_path}",
+    unless  => "${difftool} ${config_path} ${tmp_config_path}",
```

Comment:
A doc note suggesting fully qualifying the path to the diffing utility might not be a bad idea.  I can imagine issues being opened over a utility installed in `/usr/local/bin`/etc. not working.